apps/server/src/modules/saga/type/saga-step.ts

```diff
@@ -0,0 +1,39 @@
+import { EntityId } from '@shared/domain/types';
+import { StepReport } from './report';
+
+export interface StepType {
+	deleteUserData: { params: { userId: EntityId }; result: StepReport };
+	// add more, e.g.:
+	// deleteUserReference: { params: { userId: EntityId }; result: boolean };
+	// step1: { params: { value: number }; result: string };
+	// step2: { params: { message: string }; result: boolean };
+}
+
+export enum ModuleName {
+	ACCOUNT = 'account',
+	BOARD = 'board',
+	CLASS = 'class',
+	COURSE = 'course',
+	COURSE_COURSEGROUP = 'courseGroup',
+	LEARNROOM_DASHBOARD = 'dashboard',
+	FILES = 'file',
+	FILES_STORAGE = 'fileRecords',
+	LESSON = 'lessons',
+	PSEUDONYM = 'pseudonyms',
+	ROCKETCHATUSER = 'rocketChatUser',
+	ROCKETCHATSERVICE = 'rocketChatService',
```

Comment:
ROCKETCHATSERVICE module has no saga, it must be removed
it is an module (should be moved in infra) called from RocketChatUser module

Reply:
Removed.